news/203.feature

```diff
@@ -0,0 +1 @@
+Raise `PublishForbidden` exception immediately if publishing to [virtual host](https://www.rabbitmq.com/vhosts.html) is denied rather than waiting until timeout.
```

Comment:
Thanks! Unfortunately this will be inserted in a RestructuredText document so this has to RST and not Markdown... sorry!

Reply:
Fixed